### ISSUE TITLE
GitHub UI: Reduce noise

### ISSUE THREAD

**Issue by jcfr** [acil-bwh/SlicerCIP#41]
@lassoan @rjosest 

To remove the "noise" associated with the project, I suggest disabling the following unused features if it still applies:

| Go to General settings and disable the `Wiki`, `Projects` and `Discussions` tabs | If empty or unused, hide `Releases` and `Packages` section on the right side |
|--|--|
| ![GitHub: Projects & Wiki tabs](https://i.imgur.com/nMPUPVQ.png) | ![GitHub: Release & Packages sections](https://i.imgur.com/pf5VGZY.png) |

---

<sub>_This comment was adapted from my [GitHub saved replies](https://docs.github.com/en/get-started/writing-on-github/working-with-saved-replies/using-saved-replies), it is licensed under a [Creative Commons Attribution 4.0 International License][cc-by-40] and you were welcome to reuse it. [![Creative Commons License][cc-by-40-img]][cc-by-40]_</sub>

[cc-by-40]: http://creativecommons.org/licenses/by/4.0/
[cc-by-40-img]: https://i.creativecommons.org/l/by/4.0/80x15.png

**Comment by jcfr** [comment]
Note that in the context of this project, the [Wiki](https://github.com/acil-bwh/SlicerCIP/wiki) should likely remain enabled was it contains relevant details for getting SlicerCIP and getting in touch with the maintainers.

**Comment by lassoan** [comment]
I agree that these features should be turned off. I don't have admin rights to this repository, so I cannot do it myself. 

The information currently in the wiki is usually described markdown files in the repository, so if we move that content then the wiki could be turned off, too.

**Comment by rjosest** [comment]
I am OK removing those feature but I don’t understand the downside from the Slicer integration perspective.
Could you elaborate? 

Thanks

Raul

**Comment by lassoan** [comment]
It is just to simplify things.

For example, if we provide user support via the Slicer forum then it is better to turn off Discussions here, so that we don't need to monitor two user support sites, but only one. 

We don't use Releases and packages, so it is cleaner to hide them (to indicate that it is not the way how we manage releases and packages).

Projects feature is rarely used in general and this repository does not use it either. If the user clicks on Projects then an empty page is s displayed. It is better to not display the heading at all, to save the extra two clicks for users exploring the project.

Wiki on Github is meant to be a lighterweight tool for documentation. The issue is that it is too limited, for example contributors either can make changes directly (without pull request) or they cannot make changes at all. It also write redundant, since Github quite nicely renders markdown pages in the main repository. For small projects it is simpler and more consistent to keep all documentation in markdown files in the main repository.
